### DISCUSSION TITLE
Dynamic Smagorinsky

### DISCUSSION THREAD

**Discussion by Swarnali-D** [CliMA/Oceananigans.jl#3426] [Computational science!]
How to incorporate dynamic Smagorinsky-Lilly closure scheme into my Nonhydrostatic model?

**Comment by glwagner** [comment]
No flavors of a dynamic Smagorinsky closure are implemented. There have been discussions of a Lagrangian-averaged dynamic Smagorinsky but nothing has come to fruition. We're investigating whether it's better to use `closure=nothing` with `advection=WENO(order=9)`, though. See

https://agupubs.onlinelibrary.wiley.com/doi/full/10.1002/2016MS000778

cc @tomchor
